src/commands/switch/organisation.ts

```diff
@@ -0,0 +1,71 @@
+import { RequestBase } from '../../utils/request-base';
+import chalk from 'chalk';
+import ErrorResponse from '../../error-response';
+import { ICommand, IOrganisation, ISwitchOrganisationQuestionAnswers } from '../../types';
+import { CommandBuilder } from 'yargs';
+import { getProperty, processAnswers, updateLineSalableRc } from '../../utils';
+import { SWITCH_ORGANISATION_QUESTIONS } from '../../questions';
+
+const builder: CommandBuilder = {};
+
+const handler = async () => {
+  try {
+    const organisations = await RequestBase<IOrganisation[]>({
+      method: 'GET',
+      endpoint: `organisations`,
+    });
+
+    if (!organisations) {
+      // eslint-disable-next-line no-console
+      console.log(chalk.yellow(`No organisations found`));
+      return;
+    }
+
+    const orgNames = organisations?.map((org) => org.name);
```

Comment:
```suggestion
    const orgNames = organisations.map((org) => org.name);
```

Conditional chaining should be redundant because of the conditional check on `L18`